### PR TITLE
Update code requirement policy

### PR DESCRIPTION
**Pull request by wd15**
Address #183

Update code requirement policy to agree with issues raised at the
recent phase field meeting.

### CODE REVIEW

File: simulations/index.html [hunk 1]
```diff
@@ -66,7 +66,7 @@ <h5 class="sideheader">Charts</h5>
 <div class="container top-bar top-buffer">
   <div class="row small-buffer">
     <div class="col l2 m12 s12">
-      <h5 class="sideheader">Simulations</h5>
+      <h5 class="sideheader" id="simulations">Simulations</h5>
```
Comment (tkphd) [inline simulations/index.html]: id tag makes this linkable elsewhere on the site, right? And the charts tag is not yet used?

File: submit_a_new_code.md [hunk 2]
```diff
@@ -8,10 +8,19 @@ comment: >-
 
 ### Requirements
 
-The main requirements for listing a code are that the code includes an
-[open source license](http://choosealicense.com){:target="_blank"}, is
-available via a public repository and includes annotated phase field
-examples.
+The two main requirements for including a code on the front page are
```
Comment (tkphd) [inline submit_a_new_code.md]: The grammar before a list applies to each element; "... on the front page are and that" does not flow. Consider instead,
```
The two main requirements for including a code on the front page are that
1. at least one... and
2. the submitted benchmark...
```

File: submit_a_new_code.md [hunk 3]
```diff
@@ -8,10 +8,19 @@ comment: >-
...
+    }}/simulations/#simulations) is submitted for the benchmark problems using the
+    code in question (see [instructions]({{ site.baseurl
+    }}/simulations/upload))
+
+ 2. and that the submitted benchmark results are reproducible by
+    another researcher.
+
+Note that this does not limit codes to open source only, but does
+require that the necessary input files for the benchmark simulations
+are made available for other researchers in a public repository.
```
Comment (tkphd) [inline submit_a_new_code.md]: I don't love the requirements-disclaimer/explanation structure here. It might be better to reiterate our reason for collecting data, then stipulating the restrictions. Let's be clear about what we want, what we don't, and why.

```
### Call for Community Codes
This website is intended to help users judge for themselves the suitability of available codes for their purposes. Performance on the various benchmark problems are expected to form the major basis for comparison, but consideration should also be given to ease of installation, use, and comprehension of the code.

### Requirements
If you develop or use a code for phase field simulations, you are encouraged to submit it for inclusion on this site if it meets the following criteria.
1. Input files for at least one of the [benchmark problems]({{ site.baseurl }}/#benchmarks) are publicly available.
2. The program used to run the input files is accessible to members of the phase field community.
3. The program is distributed with documentation to help new users install the software and come up to speed on its use for solving phase field problems.

Note that these criteria do not exclude proprietary or closed-source software, so long as the benchmark problems can be implemented in an open format.
However, if your code is "internal use only" or lacks documentation, we cannot consider it a "community code."

### Follow-up
After your code is added to the list, please upload at least one [simulation result]({{ site.baseurl }}/simulations/#simulations) to provide users with a side-by-side comparison.
```

File: submit_a_new_code.md [hunk 4]
```diff
@@ -8,10 +8,19 @@ comment: >-
...
-examples.
+The two main requirements for including a code on the front page are
+
+ 1. that at least [one simulation result]({{ site.baseurl
+    }}/simulations/#simulations) is submitted for the benchmark problems using the
+    code in question (see [instructions]({{ site.baseurl
+    }}/simulations/upload))
+
+ 2. and that the submitted benchmark results are reproducible by
+    another researcher.
```
Comment (tkphd) [inline submit_a_new_code.md]: Let's speak plainly here. What is the criterion, exactly?

File: simulations/index.html (hunk 1 above)
Comment (wd15) [inline simulations/index.html]: Correct

File: submit_a_new_code.md (hunk 2 above)
Comment (wd15) [inline submit_a_new_code.md]: done

File: submit_a_new_code.md (hunk 4 above)
Comment (wd15) [inline submit_a_new_code.md]: I improved this with "the submitted benchmark results are reproducible by another researcher with the code in question."

File: submit_a_new_code.md (hunk 3 above)
Comment (wd15) [inline submit_a_new_code.md]: That's much nicer, I've added it as is.

File: simulations/index.html (hunk 1 above)
Comment (tkphd) [inline simulations/index.html]: :+1:

File: submit_a_new_code.md (hunk 2 above)
Comment (tkphd) [inline submit_a_new_code.md]: :+1:

File: submit_a_new_code.md (hunk 4 above)
Comment (tkphd) [inline submit_a_new_code.md]: :+1:

File: submit_a_new_code.md (hunk 3 above)
Comment (tkphd) [inline submit_a_new_code.md]: Thanks :smile: :+1:

**Comment by wd15** [comment]
@guyer, added you as a review on this as it might be useful to have another pair of eyes on this, thanks, you can decline the review if you want aldo

**Comment by tkphd** [comment]
@guyer please don't decline, we need more eyes on the public-facing content.

**Comment by wd15** [comment]
@drjamesawarren, asking for your input on this as well. Click on the "files changed" tab above on then on the "review changes" to leave any changes that you'd like to see and click on "approve" or "request changes".

**Comment by wd15** [comment]
@drjamesawarren: @tkphd wrote that and it is ambiguous, but I think it's better than being specific. It's too difficult to define easily. It has to be more than email me and I'll send you a copy of the code that I used for the benchmark problems.

**Comment by tkphd** [comment]
I think we do want to be specific. To me, accessible requires, at a minimum:
1. An executable on my hard drive, built for my OS and architecture.
    - This can come from source, a package manager, a Web link, or e-mail: it really doesn't matter.
2. Dependencies that let me run said executable.
     - This can be a list, or a conda/docker/whatever, or a packaged file from GitHub, a web page, or e-mail.
3. Guidance on how to run the executable and what to expect when I do so.
    - This should be on a public Web site.
4. License to run the program.
    - For a fee if necessary. 

Basically, I want to collect codes that were built with the intention of building a community around them. Thinking through it, a proxy for "accessible" is "has a public Web site describing how to obtain, configure, and use the program." We would like to encourage codes that help people understand phase field, and reduce the barrier to implementing new models compared to rolling one's own from scratch.
